docs/source/operations/orchestrator/deployment.rst

```diff
@@ -0,0 +1,223 @@
+
+      | Replace ``HOSTNAME`` with the hostname of your Orchestrator.
+
+   #. Delete the Certificate Signing Request:
+
+      .. code-block:: bash
+
+         rm orchestrator-cert.csr orchestrator-ca.srl
+
+#. Create a Kubernetes ConfigMap for the CA certificate:
+   
+   .. code-block:: bash
+      
+      kubectl create configmap orchestrator-tls-cacert --from-file=orchestrator-ca.crt
+
+#. Create a Kubernetes Secret for the orchestrator TLS key and certificate:
+
+   .. code-block:: bash
+      
+      kubectl create secret tls orchestrator-tls-server-pair --cert=orchestrator-tls.crt --key=orchestrator-tls.key
+
+
+
+
+We want to validate clients identity usigng mTLS so we need to enable some additional settings:
+In the same values file add:
+
+```
+orchestrator:
+  verifyClientMSPID: true
+```
+
+We enable TLS for the server:
+in te same values.yaml file
+```
+orchestrator:
+  verifyClientMSPID true
+  tls:
+    enabled: true
+```
+
+Now for this to work we need to generate some certificates.
+
+First we need to generate a CA certificate:
+for this example we will generate them manually but you can use external providers like let's encrypt for this step.
+
+> Include sample openssl config: https://github.com/Substra/orchestrator/blob/9c8106dde71ae379da0a8ae4d0bb0e8d88b68f4d/examples/tools/openssl-with-ca.cnf
+
+```
+openssl genrsa -out ca.key 2048
+openssl req -new -x509 -days 365 -sha256 -key ca.key -extensions v3_ca -config openssl-with-ca.cnf -subj "/C=FR/ST=Loire-Atlantique/L=Nantes/O=Orchestrator Root CA/CN=Orchestrator Root CA" -out ca.crt
+```
+
+You should now have these files in your current directory: `ca.crt` and `ca.key`.
+
+You can already create a ConfigMap in you cluster named `orchestrator-tls-cacert` using the command:
+```
+kubectl create configmap orchestrator-tls-cacert --from-file=ca.crt
+```
+
+> list configmap to see that it is created
+
+Generate a certificate key for the orchestrator
+```
+openssl req -newkey rsa:2048 -nodes -keyout orchestrator-tls.key -subj "/C=FR/ST=Loire-Atlantique/L=Nantes/O=Substra/CN=orc.my-corp.org" -out orchestrator-cert.csr
+```
+
+and 
+```
+openssl x509 -req -days 365 -in orchestrator-cert.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out orchestrator-tls.crt -extfile <(printf "subjectAltName=DNS:orc.my-corp.org")
+```
+
+Then we need to create a kubernetes secret containing these informations
+
+```
+kubectl create secret tls orchestrator-tls-server-pair --cert=orchestrator-tls.crt --key=orchestrator-tls.key
+```
+
+Now you can run an orchestrator that is secured
+
+```
+helm install my-orchestrator substra/orchestrator --version 7.4.3 --values orchestrator-values.yaml
+```
+
+
+```
+GODEBUG=x509sha1=1 grpcurl --cacert ca.crt --rpc-header 'mspid: MyOrg1MSP' --rpc-header 'channel: mychannel' --rpc-header 'chaincode: mycc' orchestrator.install-guide.cg.owkin.tech:443 describe
```

Comment:
Is this still needed?
```suggestion
grpcurl --cacert ca.crt --rpc-header 'mspid: MyOrg1MSP' --rpc-header 'channel: mychannel' --rpc-header 'chaincode: mycc' orchestrator.install-guide.cg.owkin.tech:443 describe
```

Reply:
It's the non-formatted copy/paste part of the doc.